Survey/Survey/ContentView.swift

```diff
@@ -1,13 +1,22 @@
 import SwiftUI
 
 struct ContentView: View {
-    var body: some View {
-        SurveyView()
-    }
-}
+    @StateObject private var viewModel = SurveyViewModel()
 
-struct ContentView_Previews: PreviewProvider {
-    static var previews: some View {
-        ContentView()
+    var body: some View {
+        VStack {
+            if viewModel.inquiries.isEmpty {
+                EmptyView()
+            } else {
+                ForEach(viewModel.inquiries) { inquiry in
+                    SurveyView(question: inquiry.question)
```

Comment:
I'm a little confused here -- do we expect to have multiple inquiries?

Reply:
In case, admin would like to show a few questions, we can do it. But, for now it gonna be only one.